src/gatsby-node.js

```diff
@@ -107,14 +107,31 @@ const replaceImage = async ({
   // find the full size image that matches, throw away WP resizes
   const parsedUrlData = parseWPImagePath(thisImg.attr("src"))
   const url = parsedUrlData.cleanUrl
+  let imageNode;
 
-  const imageNode = await downloadMediaFile({
-    url,
-    cache,
-    store,
-    createNode,
-    createNodeId,
-  })
+  try{
```

Comment:
Thanks so much for this! It looks great. Would you mind adding a couple comments as to why we have to do this and what the second download is for?
I can also add them if you'd like, just don't want to mess with your PR :) Let me know